cli/features/tutorial.feature

```diff
@@ -266,7 +266,9 @@ Feature: Following the tutorial
       """
     When running "exo setup" in this application's directory
     And starting "exo run" in this application's directory
-    And waiting until I see "all services online" in the terminal
+    # TODO: change back to 'And waiting until I see "all services online" in the terminal'
+    # when online text monitoring is implemented
+    And waiting until I see "HTML server is running" in the terminal
```

Comment:
I thought this was a better solution than waiting a set amount of time. Open to alternative suggestions.